src/styles/style.css

```diff
@@ -0,0 +1,132 @@
+@import './utils/reset.css';
```

Comment:
Using `@import` in CSS is generally discouraged due to performance issues. Consider using `@import` in SCSS or linking stylesheets directly in HTML.